spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -2,16 +2,18 @@
 
 describe "View authentication requests for a username" do
   context "with results" do
-    let(:user) { create(:user, :confirmed, :with_organisation) }
     let(:username) { "AAAAAA" }
+    let(:organisation) { create(:organisation) }
```

Comment:
This spec feels like it still has "too much" setup, but I think that's going to be pretty tricky to fix. 

I'm happy to move forward with it like this for the moment, but I want to do some work on it - possibly these specs in general, as our models are getting complex - if we can find time.

Reply:
Yeah I agree. Coming from the outside and learning all this from scratch I feel it was pretty complex to set up so maybe in the future we can refactor the specs to be more 'test friendly' @kjdchapman